src/soc/ibm/power9/bootblock.c

```diff
@@ -1,5 +1,7 @@
 /* SPDX-License-Identifier: GPL-2.0-only */
 
+void bootblock_soc_early_init(void);
```

Comment:
What for? bootblock_soc_early_init is declared in bootblock_common.h

Reply:
It is fixed in Krystian's MR to this branch. [https://github.com/3mdeb/coreboot/pull/33/files](https://github.com/3mdeb/coreboot/pull/33/files)